server/src/main/java/org/opensearch/threadpool/ThreadPool.java

```diff
@@ -183,7 +183,7 @@ public static ThreadPoolType fromType(String type) {
         map.put(Names.SYSTEM_WRITE, ThreadPoolType.FIXED);
         map.put(Names.TRANSLOG_TRANSFER, ThreadPoolType.SCALING);
         map.put(Names.TRANSLOG_SYNC, ThreadPoolType.FIXED);
-        map.put(Names.REMOTE_PURGE, ThreadPoolType.SCALING);
+        map.put(Names.REMOTE_PURGE, ThreadPoolType.FIXED);
```

Comment:
Is this creating overhead that will be unused on some cluster configurations?

Reply:
Yes, but the overhead is quite minimal , given the size of the pool.

Reply:
Can you help me understand how this pool is used? I don't have a sense for what 'minimal' means as some customers operate OpenSearch on very resource constrained systems such as EC2's T3 instance types.

Reply:
This pool is used for async deletions related to remote store . 

> I don't have a sense for what 'minimal' means as some customers operate OpenSearch on very resource constrained systems such as EC2's T3 instance types.

Actually on a domain with no remote store, there is not going to be any overhead . Fixed Size threadpool creates `SizeBlockingQueue` with some fixed  size . With no remote purges happening , it will not create any overhead as the queue will not have any items in the first place. 

For remote store enabled clusters, this will help limit the memory impact of this queue by limiting its size.